tools/update_all.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -x -o errexit
+
+tools/generate.py Adw 1 -o src/gi-stubs/repository/Adw.pyi
```

Comment:
Maybe we should store the ABI versions in a JSON file to make it cleaner?

Reply:
yes, sounds like a good idea.

Json file with name, abi version and path, and then simply iterate with a script through that.
also we can call back and isort after each iteration on the current output path. And on error simply continue.

this could also be a python script, not sure how easy it is with bash to load and iterate over json.

Instead of a json we could also simply maintain a dict or list in a python file, if the update script would be written in python.

Reply:
Converted it to Python.